project/src/main/java/com/google/sps/servlets/DatamuseServlet.java

```diff
@@ -36,27 +52,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
                 con.setRequestMethod("GET");
                 con.setRequestProperty("Content-Type", "application/json");
 
-                StringBuilder output;
+                StringBuilder output = new StringBuilder();
                 BufferedReader input = new BufferedReader(new InputStreamReader(con.getInputStream()));
```

Comment:
These are only used inside `addToString` function, so just create them there instead of passing them as parameters.